README.md

```diff
@@ -158,6 +158,12 @@ The above command pulled the existing data out of Route53 and placed the results
 | [TinyDNSSource](/octodns/source/tinydns.py) | A, CNAME, MX, NS, PTR | No | read-only |
 | [YamlProvider](/octodns/provider/yaml.py) | All | Yes | config |
 
+#### Notes
+
+* ALIAS support varies a lot fromm provider to provider care should be taken to verify that your needs are met in detail.
+   * Dyn's UI doesn't allow editing or view of TTL, but the API accepts and stores the value provided, this value does not appear to be used when served
+   * Dnsimple's API throws errors when TTL is modified, but it can be edited in the UI and seems to be used when served, there's also a secondary TXT record created alongside the ALIAS that octoDNS ignores
```

Comment:
Double checked this and actually I fixed the error that was happening here and forgot to update the README. So that part doesn't apply any longer. I did leave the bits about DNSimple seeming to obey the ALIAS ttl and creating a ride along record. Thanks for catching this.